build/yamls/base/crds.yml

```diff
@@ -475,6 +475,10 @@ spec:
                       podSelector:
                         type: object
                         x-kubernetes-preserve-unknown-fields: true
+                      # TODO, Uncomment to support applying NP on ExternalEntity.
```

Comment:
Could we add it when we support ExternalEntity? It looks strange to uses have no knowledge on ExternalEntity.

Reply:
Agree that this does not seem to be relevant to the PR

Reply:
sure will remove